go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/uc-cdis/arborist
+
+go 1.17
+
+require (
+	github.com/gorilla/handlers v1.5.1
+	github.com/gorilla/mux v1.8.0
+	github.com/jmoiron/sqlx v1.3.4
+	github.com/lib/pq v1.10.3
+	github.com/stretchr/testify v1.7.0
+	github.com/uc-cdis/go-authutils v0.0.0-20211013194331-22cfd18557de
```

Comment:
why not a new release?

Reply:
I pushed it, so Jenkins can test the latest changes, and because version is not available for some reasons [here](https://proxy.golang.org/github.com/uc-cdis/go-authutils/@v/v0.1.1.info) or [here](https://pkg.go.dev/github.com/uc-cdis/go-authutils?tab=versions).

I'll take a look tomorrow on what's missing for the version.